### PR TITLE
Changes as requested - made tests more generic

### PR DESCRIPTION
**Pull request by DanWaldman**
- made tests generic
- changed function names
- fixed typos

### CODE REVIEW

**Review by DanielItkes — APPROVED** [review]

**Review by shakedhagag — COMMENTED** [review]
Reviewed.

**Review by razashtar93 — APPROVED** [review]

**Comment by Omeramsc** [comment]
Please add information to the PR description, and link an issue. 
do note that I for instance am not aware of what PR was merged or what changes were requested but Arie, so the information provided unfortunately means nothing for me...

The PR title and description should clearly reflect the changes in the PR, not the reason for them (that however *can* be *added* as additional information to the description)

File: AcadeMeData/tests.py [hunk 1]
```diff
@@ -47,51 +35,56 @@ def test_get_user(self):
 
 @pytest.mark.django_db
 class TestUniversityModel:
-    def university_example(self):
-        university = University(university_id=1, name='Reichman University', location="Herzlia",
-                                description="A nice place")
+    def generate_university(self, university_id=1, name='Reichman University', location="Herzlia",
```
Comment (Yarboa) [inline AcadeMeData/tests.py:38]: Please move the generate_ create_ data methods to fixtures

**Comment by DanWaldman** [comment]
All is fixed and updated in #69

**Pull request closed without merging**